### PR TITLE
Browse cleanup

### PR DESCRIPTION
**Pull request by romeovs**
Some clean up for the `metabase/browse` folder, by splitting up the components according to domain (eg. Models, Metrics, Databases, etc.).

This makes it code of different domains is not mixed in common utils files.

### CODE REVIEW

File: enterprise/frontend/src/metabase-enterprise/content_verification/models.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+const USER_SETTING_KEY = "browse-filter-only-verified-models";
+
+export function getDefaultModelFilters(state: State): ModelFilterSettings {
+  return {
+    verified: getSetting(state, USER_SETTING_KEY) ?? false,
+  };
+}
+
+// This component is similar to the ModelFilterControls component from ./ModelFilterControls.tsx
+// merging them might be a good idea in the future.
```
Comment (rafpaf) [inline enterprise/frontend/src/metabase-enterprise/content_verification/models.tsx:22]: Should this file be called ModelFilterControls?

Is this comment still true?

File: enterprise/frontend/src/metabase-enterprise/content_verification/models.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  );
+
+  return (
+    <Popover position="bottom-end">
+      <Popover.Target>
+        <Button
+          p="sm"
+          lh={0}
+          variant="subtle"
+          color="text-dark"
```
Comment (rafpaf) [inline enterprise/frontend/src/metabase-enterprise/content_verification/models.tsx:46]:
```suggestion
          color="var(--mb-color-text-dark)"
```

Comment (rafpaf) [reply enterprise/frontend/src/metabase-enterprise/content_verification/models.tsx:46]: if you want to update this to our new way, in passing

**Review by rafpaf — APPROVED** [review]

File: enterprise/frontend/src/metabase-enterprise/content_verification/models.tsx (hunk 2 above)
Comment (ranquild) [reply enterprise/frontend/src/metabase-enterprise/content_verification/models.tsx]: I didn't know it's a new way. It should be safe to use mantine theme colors by default. Anything changed?

**Pull request merged**